verify_test_type/README.md

```diff
@@ -0,0 +1,37 @@
+<h1 align="center">Verify Test Type</h1>
+
+<p align="center">Script Purpose: Ensure the identification of the test type (e.g., ID, OD, NOD, etc.) for new tests discovered in a repository. </p>
+
+## Steps to Run the Script locally
+
+#### 1. Go to the root directory
+
+Go to the root directory of the repository which has the newly found flaky test.
+
+#### 2. Download the script in your repository
```

Comment:
Why not just `git clone ...idoft`, especially if you're already changing `pr-data.csv` or other files in the repo?

Reply:
We only need the script file and not the entire repository so its better to just add the verify_test script to the repo.

Reply:
But installing `wget` ends up more challenging than just `git clone`. Also, why include here instructions for downloading `wget` when people can download that file/script in any other way they want.

Reply:
I will remove the wget and chmod commands, and instead mentioning that users can download that file/script in any way they want.